relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/middle/wind/Futaba.kt

```diff
@@ -0,0 +1,99 @@
+package xyz.qwewqa.relive.simulator.core.presets.dress.middle.wind
+
+import xyz.qwewqa.relive.simulator.core.presets.condition.MiddleOnlyCondition
+import xyz.qwewqa.relive.simulator.core.presets.condition.NormalDamageOnlyCondition
+import xyz.qwewqa.relive.simulator.core.stage.actor.*
+import xyz.qwewqa.relive.simulator.core.stage.autoskill.new
+import xyz.qwewqa.relive.simulator.core.stage.buff.*
+import xyz.qwewqa.relive.simulator.core.stage.loadout.Dress
+import xyz.qwewqa.relive.simulator.core.stage.passive.*
+import xyz.qwewqa.relive.simulator.stage.character.Character
+import xyz.qwewqa.relive.simulator.stage.character.DamageType
+import xyz.qwewqa.relive.simulator.stage.character.Position
+
+val HondaFutaba = Dress(
+    name = "Honda Futaba 9.8",
```

Comment:
For maxed cards leave out the rank since that's the default.

Reply:
Done.